.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout working branch
+      uses: actions/checkout@v1
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install latest versions of pip, setuptools and wheel
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+
+    - name: Cache pip
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: pip-${{ hashFiles('setup.cfg') }}
```

Comment:
I am checking against `setup.cfg` as the dependencies are defined here as we don't have a `requirements.txt`.

Maybe we should include `setup.py` as well? Because others might find useful to define package dependencies there.

Reply:
:+1: to both